services/venice-controller/src/main/java/com/linkedin/venice/controller/VeniceHelixAdmin.java

```diff
@@ -793,6 +794,14 @@ private synchronized void connectToControllerCluster() {
     }
     controllerClusterKeyBuilder = new PropertyKey.Builder(tempManager.getClusterName());
     helixManager = tempManager;
+
+    VeniceControllerClusterConfig controllerConfig = multiClusterConfigs.getCommonConfig();
+    List<String> instanceTagList = controllerConfig.getControllerInstanceTagList();
```

Comment:
Since instance tag is an instance-level config, we should add the `getInstanceTagList` function in `multiClusterConfigs` and handle the part about fetching from common configs there?